### ISSUE TITLE
LC2-21: [FATAL] Unable to generate Computer mount.: class java.io.FileNotFoundException:

### ISSUE THREAD

**Issue by HydrantHunter** [PC-Logix/LanteaCraft#425]
### Submission checklist

- [X] Check that you are running the latest recommended version of LanteaCraft.
- [X] Check that the issue isn't a known problem already (https://github.com/PC-Logix/LanteaCraft/issues).
- [ ] Check that you can repeat this issue with just LanteaCraft installed.  NOTE: ComputerCraft also required.

### Expected behavior

No expected behavior beyond not generating the error.

### Actual behavior

When loading my test map, I get the above error in the logs.  Upon shutdown, the following is generated in the logs...
[FATAL] Unable to garbage collect Computer mount.: class java.lang.RuntimeException: You didn't mount this location

From a user perspective, this doesn't seem to interfere with the operation of CC & LanteaCraft within the game.

### Steps to reproduce

No special steps required that I'm aware of - simply load a map with LC2-21 stargates and ComputerCraft 1.75.  Have at least one of the computers wrap a stargate.

### System information

- MinecraftForge version: 10.13.4.1558
- LanteaCraft version: LC2-21
- Logs: [LC2 game.log](http://pastebin.com/Wuid7WiH)

**Comment by AfterLifeLochie** [comment]
The underlying problem is that `ComputerCraftScuffMount` is not correctly generating the system mount path:

```
[DEBUG] Working path: file:/D:/Games/Sim/Minecraft/MultiMC/instances/LC2/minecraft/mods/LanteaCraft-1.7.10-21.jar
[FATAL] Unable to generate Computer mount.: class java.io.FileNotFoundException: file:\D:\Games\Sim\Minecraft\MultiMC\instances\LC2\minecraft\mods\LanteaCraft-1.7.10-21.jar (The filename, directory name, or volume label syntax is incorrect)
java.io.FileNotFoundException: file:\D:\Games\Sim\Minecraft\MultiMC\instances\LC2\minecraft\mods\LanteaCraft-1.7.10-21.jar (The filename, directory name, or volume label syntax is incorrect)
```

The path is correct, except the system is still appending the `file://` prefix.

**Comment by AfterLifeLochie** [comment]
Should be fixed in commit bb3eeb6ab04b0ebda1dd60b0ed85259e20b54072.
